homeassistant/components/tuya/alarm_control_panel.py

```diff
@@ -0,0 +1,149 @@
+"""Support for Tuya Alarm."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+
+from tuya_iot import TuyaDevice, TuyaDeviceManager
+
+from homeassistant.components.alarm_control_panel import (
+    AlarmControlPanelEntity,
+    AlarmControlPanelEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeAssistantTuyaData
+from .base import TuyaEntity
+from .const import DOMAIN, TUYA_DISCOVERY_NEW, DPCode
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
There is a central logger in the Tuya integration that can be imported from `.const`